CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
 # Developing UberFx
```

Comment:
@Lupie - we need to call the project the same thing everywhere: Fx

Can you make a pass at all the docs in this repo to make sure we aren't calling it UberFx anymore?